crates/sequencer/src/commitment/service.rs

```diff
@@ -22,7 +22,7 @@ use tokio::sync::{mpsc, oneshot};
 use tracing::{debug, error, info, instrument, warn};
 
 use super::controller::CommitmentController;
-use crate::metrics::SEQUENCER_METRICS;
+use crate::metrics::SEQUENCER_METRICS as sm;
```

Comment:
```suggestion
use crate::metrics::SEQUENCER_METRICS as SM;
```
I am not sure if clippy is going to whine about this but since we're importing a constant / static, we should capitalize it.